packages/@lwc/engine-core/src/framework/vm.ts

```diff
@@ -323,6 +323,11 @@ export function getAssociatedVM(obj: VMAssociable): VM {
     return vm!;
 }
 
+export function isConnected(obj: VMAssociable): boolean {
+    const vm = getAssociatedVM(obj);
+    return vm.state === VMState.connected;
```

Comment:
since this is going away with node-reactions, this seems like the wrong abstraction.